server/config_test.go

```diff
@@ -15,6 +15,8 @@ func TestLoadUsers(t *testing.T) {
 		expUsers []user.UserWithRemoteIdentities
 		// userid -> plaintext password
 		expPasswds map[string]string
+
+		wantErr bool
```

Comment:
Because I want this case to cause an error in `loadUsersFromReader` I don't actually care what error it causes.

Without the check above this test doesn't pass, with it it does. Do I need to peg the test to the exact error?